tt_metal/common/tt_backend_api_types.hpp

```diff
@@ -156,6 +156,7 @@ inline std::ostream& operator<<(std::ostream& os, const RISCV& riscv) {
         case RISCV::TRISC0: os << "TRISC0"; break;
         case RISCV::TRISC1: os << "TRISC1"; break;
         case RISCV::TRISC2: os << "TRISC2"; break;
+        case RISCV::ERISC: os << "ERISC"; break;
```

Comment:
Should we use magic_enum here and replace the switch statement?

Reply:
Leaving it for now since I think we'll have to rework/standardize a bunch of these soon